app/ModelSerializers/Summit/Presentation/AdminVoteablePresentationSerializer.php

```diff
@@ -0,0 +1,55 @@
+<?php namespace ModelSerializers;
+use models\summit\Presentation;
+
+/**
+ * Copyright 2022 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+/**
+ * Class AdminVoteablePresentationSerializer
+ * @package ModelSerializers
+ */
+class AdminVoteablePresentationSerializer extends AdminPresentationSerializer
+{
+    /**
+     * @return string
+     */
+    protected function getSpeakersSerializerType():string{
+        return SerializerRegistry::SerializerType_Private;
+    }
+
+    /**
+     * @param null $expand
+     * @param array $fields
+     * @param array $relations
+     * @param array $params
+     * @return array
+     */
+    public function serialize(
+        $expand = null, array $fields = array(), array $relations = array(), array $params = array())
+    {
+        $presentation = $this->object;
+        if (!$presentation instanceof Presentation) return [];
+
+        if (!count($relations)) $relations = $this->getAllowedRelations();
+
+        $values = parent::serialize($expand, $fields, $relations, $params);
+
+        $beginVotingDate = $params['begin_attendee_voting_period_date'] ?? null;
+        $endVotingDate = $params['end_attendee_voting_period_date'] ?? null;
+
+        $values['voters'] = $presentation->getVoters($beginVotingDate, $endVotingDate);
```

Comment:
here you should fo a for each and call the proper serializer
like foreach($presentation->getVoters($beginVotingDate, $endVotingDate) as $voter){
$res[] =  SerializerRegistry::getInstance()->getSerializer($voter)->serialize(AbstractSerializer::filterExpandByPrefix($expand, $relation));
}

but please just do it is expand contains the proper expand ( in this case voters)
if not u should return the voters id

Reply:
done as requested